Apps/DiceRoller/DiceRoller/DiceDataController.m

```diff
@@ -27,108 +27,5 @@ - (void)drawRect:(CGRect)rect
 }
 */
```

Comment:
Same commented code idea here.  If you don't need this commented out `drawRect:(CGRect)rect` function in `DiceDataController.m`, please remove it.